core/urls.py

```diff
@@ -78,3 +82,13 @@ class PaginatedSitemap(GenericSitemap):
         ]
     )
     urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+
+# Wagtail urls
+urlpatterns.extend(
+    [
+        path("cms/", include(wagtailadmin_urls)),
+        path("documents/", include(wagtaildocs_urls)),
+        path("", include(wagtail_urls)),
```

Comment:
à la racine ? je suppose que `wagtail_urls` défini des urls qui ne vont pas surcharger la racine mais instinctivement je me méfie

Reply:
alors on pourrait namespacer, genre tout mettre sous une url `pages` par exemple.
Je te rejoins sur le risque de ne pas namespacer, le risque n'est pas nul, mais je l'ai volontairement ajouté en dernier afin qu'une page wagtail n'override pas une route définie dans le projet.

Tu imaginais quels problèmes potentiels ?
Est-ce qu'un commentaire dans le fichier permettrait d'éviter ces problèmes ?

```suggestion
        # This URL should not be move above other "" paths in order to prevent
        # Wagtail pages' slugs to override internal routes defined in the project
        path("", include(wagtail_urls)),
```